lib/binance_client/models/order.rb

```diff
@@ -0,0 +1,18 @@
+module BinanceClient
+  class Order
+    attr_reader :symbol
+    attr_reader :executed_quantity
+    attr_reader :cummulative_quote_quantity
+    attr_reader :order_id
+    attr_reader :side
+
+    def initialize(hash)
+      @symbol = hash["symbol"]
+      @executed_quantity = hash["executedQty"]
+      @cummulative_quote_quantity = hash["cummulativeQuoteQty"]
+      @order_id = hash["orderId"]
```

Comment:
I suggest typecasting these already so that it doesn't need to be done in app. There's no need to ever want the String version anyway of `executedQty`, for example.

Reply:
Btw, the typecast is better done in the reader method, and not during initialization because a) if the value isn't called, then it's extra CPU cycles, b) if there's a problem typecasting something during initiazation, then the whole process blows up